PolyEngine/Core/Core.vcxproj

```diff
@@ -130,9 +130,7 @@
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <ClCompile Include="Src\BaseObject.cpp" />
```

Comment:
Because it was empty (baring the single include) and useless. The implementation is wholly contained in the header file (maybe we should move template implementations to .tpp files?)